lib/models/News.js

```diff
@@ -44,7 +44,7 @@ export default class News extends WorldstateObject {
      * The news's image link
      * @type {string}
      */
-    this.imageLink = wfCdn(data.ImageUrl || cdn('img/news-placeholder.png'));
+    this.imageLink = data.ImageUrl || cdn('img/news-placeholder.png');
```

Comment:
yeah, no

Reply:
Care to elaborate?

Reply:
lease look at the latest commit on master